helm/charts/os2iot-backend/templates/deployment.yaml

```diff
@@ -55,16 +55,16 @@ spec:
               value: os2iot-kafka-svc
             - name: KAFKA_PORT
               value: '9092'
-            - name: CHIRPSTACK_APPLICATION_SERVER_HOSTNAME
-              value: chirpstack-application-server-svc
-            - name: CHIRPSTACK_NETWORK_SERVER
-              value: chirpstack-network-server-svc
+            - name: CHIRPSTACK_HOSTNAME
+              value: localhost
+            - name: CHIRPSTACK_PORT
+              value: 8084
             - name: CS_MQTT_HOSTNAME
               value: mosquitto-svc
             - name: CS_MQTT_PORT
               value: '1883'
             - name: BACKEND_BASEURL
-              value: 'https://test-os2iot-backend.os2iot.dk'
+              value: 'http://localhost:3000'
```

Comment:
Probably fix

Reply:
To what? test-os2iot-backend seemed so weird as well, hah. But might be better.  I will change back.